tests/bson-deserialize-typewrapped-binary-001.phpt

```diff
@@ -0,0 +1,105 @@
+--TEST--
+BSON serialization: type wrapped Binary
```

Comment:
I'm already using the same thing for all the root/document BSON serialization tests.